README.md

```diff
@@ -128,6 +128,10 @@ for more information about extending configuration files.
 - [eslint-plugin-mocha](https://github.com/lo1tuma/eslint-plugin-mocha)
 - [eslint-plugin-jasmine](https://github.com/tlvince/eslint-plugin-jasmine)
 
+## Related Projects
+
+- [eslint-plugin-jest-formatting](https://github.com/dangreenisrael/eslint-plugin-jest-formatting)
```

Comment:
could you add a description here as well? 🙂

Reply:
Sure

Reply:
@SimenB Done